ocaml/idl/schematest.ml

```diff
@@ -3,7 +3,7 @@ let hash x = Digest.string x |> Digest.to_hex
 (* BEWARE: if this changes, check that schema has been bumped accordingly in
    ocaml/idl/datamodel_common.ml, usually schema_minor_vsn *)
 
-let last_known_schema_hash = "428caff23cdb969c59a9960beefd7bb6"
+let last_known_schema_hash = "88ce88a5c467ddfb8e65d9bdacd3d67a"
```

Comment:
Not a big deal, but I suggest updating this as part of the commit that changes the datamodel, so we get unit tests pass on all commits

Reply:
I was previously asked to put this into a separate commit (it was part of the datamodel commit initially) because it'd need changing closer to merge into master.